playbooks/tests/tasks/aodh.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: test aodh
+  hosts: controller[0]
+  vars:
+    event_alarm: test_event_alarm
+  tasks:
+  - name: Get image id for event alarm creation
+    shell: . /root/stackrc; glance image-list | grep cirros | awk '{print $2}'
```

Comment:
Or even better, use the `os_image_facts` module to get details about the particular image.